.github/ISSUE_TEMPLATE/4-constraint-work.yaml

```
@@ -0,0 +1,112 @@
+name: Create or Revise a Constraint
+description: Define work to be performed specifically related to the creation, refinement, or decommissioning of constraints.
+labels: ["enhancement"]
+body:
+  - type: textarea
+    id: user-story
+    attributes:
+      label: Constraint Task
+      description: Identify the constraint 
+      placeholder: |
+        Consistent with the parent/tracked-by issue, this constraint work focuses on ...
+    validations:
+      required: true
+
+  - type: textarea
+    id: intended-outcome
+    attributes:
+      label: Intended Outcome
+      description: Describe the reason for the constraint 
+      placeholder: |
+        Ensure the {*content*} is ...
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: OSCAL Content Details
+      
+  - type: dropdown
+    attributes:
+      label: Syntax Type
+      options:
+      - This is required core OSCAL syntax.
+      - This is optional core OSCAL syntax.
+      - This is a FedRAMP extension.
+      - This is a mix of required, optional, and/or extended syntax.
```

Can we add a not sure for more junior staff/community members who may not know given it is required?

```suggestion
      - This is a mix of required, optional, and/or extended syntax.
      - Not sure, can maintainers help me choose?
```